.github/CODEOWNERS

```diff
@@ -422,7 +422,7 @@
 # ServiceOwners:                                                   @a-t-mason @ganzee @manuaery
 
 # ServiceLabel: %Data Factory
-# ServiceOwners:                                                   @shawnxzq @lmy269
+# ServiceOwners:                                                   @ruowan @Frey-Wang @shawnxzq
```

Comment:
- The account `ruowan` needs to have the Azure organization made public on their profile.
- The account `Frey-Wang` needs to have the Microsoft and Azure organizations made public on their profile.

Both accounts are confirmed to have "write" permissions already.

Reply:
The two account profile is updated.

Reply:
`Frey-Wang` has made their `Azure` membership public.   They still need to make their `Microsoft` membership public.